Docs - clarify datasource API sources (#11489)
apache/druid: docs/operations/api-reference.md

> Returns a list of datasource names found in the cluster.
> 
> This method reads from the server view held by the coordinator that is updated every [`druid.coordinator.period`](../configuration/index.html#coordinator-operation), whereas `/druid/coordinator/v1/metadata/datasources` reads from metadata database.

Reviewer: ```suggestion
Returns a list of datasource names found in the cluster as seen by the coordinator. This view is updated every [`druid.coordinator.period`](../configuration/index.html#coordinator-operation).
```
